app/matplotlib.ipynb

```diff
@@ -194,21 +195,21 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 10,
-   "outputs": [],
    "source": [
     "labels = [\"Product A\", \"Product B\", \"Product C\", \"Product D\"]\n",
     "market_share = [30, 20, 15, 35]\n",
     "\n",
-    "# write your code here"
+    "plt.pie(market_share, labels=labels, autopct=\"%10.0f%%\", startangle=0)\n",
+    "\n",
+    "plt.title(\"Pie plot\")\n",
+    "plt.xlabel(\"X axis\")\n",
+    "plt.ylabel(\"Y axis\")"
```

Comment:
The x and y labels are unnecessary for a pie plot and should be removed to clean up the visualization.